src/internal/components/autosuggest-input/index.tsx

```diff
@@ -209,7 +209,7 @@ const AutosuggestInput = React.forwardRef(
       }
     };
 
-    const expanded = open && (dropdownExpanded ?? !!dropdownContent);
+    const expanded = open && dropdownExpanded;
```

Comment:
aria-expanded need to be set to false if there are no real options present (but there might be a special "Use" option still).